cmd/kubebuilder/create/config/gen.go

```diff
@@ -185,15 +176,104 @@ func getDeployment(p *parse.APIs) string {
 	return string(s)
 }
 
+func getStatefuleSet(p *parse.APIs) string {
+	var replicas int32 = 1
+	labels := addLabels(map[string]string{
+		"control-plane": "controller-manager",
+	})
+	statefulset := appsv1.StatefulSet{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "apps/v1",
+			Kind:       "StatefulSet",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%v-controller-manager", name),
+			Namespace: fmt.Sprintf("%v-system", name),
+			Labels:    labels,
+		},
+		Spec: appsv1.StatefulSetSpec{
+			ServiceName: fmt.Sprintf("%v-controller-manager-service", name),
+			Replicas:    &replicas,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: labels,
+			},
+			Template: getPodTemplate(labels),
+		},
+	}
+
+	s, err := yaml.Marshal(statefulset)
+	if err != nil {
+		glog.Fatalf("Error: %v", err)
+	}
+	return string(s)
+
+}
+
+func getStatefuleSetService(p *parse.APIs) string {
+	labels := addLabels(map[string]string{
+		"control-plane": "controller-manager",
+	})
+	statefulsetservice := corev1.Service{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Service",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%v-controller-manager-service", name),
+			Namespace: fmt.Sprintf("%v-system", name),
+			Labels:    labels,
+		},
+		Spec: corev1.ServiceSpec{
+			Selector:  labels,
+			ClusterIP: "None",
+		},
+	}
+
+	s, err := yaml.Marshal(statefulsetservice)
+	if err != nil {
+		glog.Fatalf("Error: %v", err)
+	}
+	return string(s)
+
+}
+
+func getPodTemplate(labels map[string]string) corev1.PodTemplateSpec {
+	var terminationPeriod int64 = 10
+	return corev1.PodTemplateSpec{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: labels,
+		},
+		Spec: corev1.PodSpec{
+			TerminationGracePeriodSeconds: &terminationPeriod,
+			Containers: []corev1.Container{
+				{
+					Name:    "controller-manager",
+					Image:   controllerImage,
+					Command: []string{"/root/controller-manager"},
+					Args:    []string{"--install-crds=false"},
+					Resources: corev1.ResourceRequirements{
+						Requests: map[corev1.ResourceName]resource.Quantity{
+							"cpu":    resource.MustParse("100m"),
```

Comment:
Should these hard-coded values (e.g. "100m") be more configurable?

Reply:
This is fine for now.  Can ask Ken what reasonable defaults should be.  Can use kinflate to override them.